packages/mdc-switch/mdc-switch.scss

```diff
@@ -13,3 +13,117 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+
+@import "@material/elevation/mixins";
+@import "@material/rtl/mixins";
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+@import "./functions";
+@import "./mixins";
+@import "./variables";
+
+.mdc-switch {
+  @include mdc-switch-toggled-on-track-color($mdc-switch-baseline-theme-color);
+  @include mdc-switch-toggled-on-thumb-color($mdc-switch-baseline-theme-color);
+  @include mdc-switch-toggled-off-track-color($mdc-switch-toggled-off-track-color);
+  @include mdc-switch-toggled-off-thumb-color($mdc-switch-toggled-off-thumb-color);
+  @include mdc-switch-toggled-off-ripple-color($mdc-switch-toggled-off-ripple-color);
+
+  display: inline-block;
+  position: relative;
+  outline: none;
+  user-select: none;
+}
+
+.mdc-switch__native-control {
+  @include mdc-rtl-reflexive-position(left, -24px);
+
+  position: absolute;
+  width: $mdc-switch-native-control-width;
+  height: $mdc-switch-tap-target-size;
+  opacity: 0;
+  cursor: pointer;
+}
+
+.mdc-switch__track {
+  box-sizing: border-box;
+  width: $mdc-switch-track-width;
+  height: $mdc-switch-track-height;
+  transition:
+    mdc-switch-transition(opacity),
+    mdc-switch-transition(background-color),
+    mdc-switch-transition(border-color);
+  border: 1px solid;
+  border-radius: $mdc-switch-track-height / 2;
+  opacity: .38;
+}
+
+.mdc-switch__thumb-underlay {
+  @include mdc-rtl-reflexive-position(left, -14px);
+  @include mdc-ripple-surface();
+  @include mdc-ripple-radius-unbounded;
+  @include mdc-states(
+    $mdc-switch-baseline-theme-color,
+    $has-nested-focusable-element: true);
+
+  display: flex;
+  position: absolute;
+  // Ensures the knob is centered on the track.
+  top: -17px;
```

Comment:
Mathematically this is `-($mdc-switch-track-height / 2 + $mdc-switch-thumb-diameter / 2)` :)

The distance between the center of the track and the thumb relative to its original position.

Reply:
I can switch to that to be more clear!